composer.json

```diff
@@ -24,15 +24,12 @@
     ],
     "require": {
         "php": ">=5.4.4",
-        "akeneo/pim-community-dev": "dev-master@dev",
         "fzaninotto/faker": "1.4.0"
     },
     "require-dev": {
+        "akeneo/pim-community-dev": "dev-master@dev",
```

Comment:
This is a Bundle for pim-community. You can't use without pim-community. We added it in require-dev for developers who wants to run specs.